app/views/content/life-as-a-teacher/teaching-as-a-career/career-progression.md

```diff
@@ -48,81 +55,74 @@ keywords:
+### Become a mentor
 
-## How can teachers increase their salary?
+You can become a mentor for early career teachers (ECTs). In this role, you’ll receive training on mentorship and coach a newly qualified teacher at the start of their career.
 
-<strong>Fran:</strong> If you have qualified teacher status (QTS) and work in a maintained school, a school funded by the local authority, your salary is reviewed every year. Most teachers on the main pay scale get an annual pay rise but it may depend on your school’s performance data to prove you’re completing whole school and individual targets. So you could potentially go from a starting salary of $salaries_starting_min$ to at least $salaries_fiveyears_max$, or more if you’re in London, after 5 years.
+## Salary progression
 
+Qualified teachers get a minimum starting salary of $salaries_qualifiedteacher_englandmin$ (or more in London). Your salary will be reviewed every year, with most teachers moving up the pay range annually. This will depend on your school’s performance management arrangements. 
 
-As well as helping you understand the demands of a more senior role, a teaching and learning responsibility (TLR) payment could make a big difference to your pay. You could earn up to an extra $salaries_tlr_max$ on top of your salary.
+A typical teacher could earn at least $salaries_fiveyears_max$ (or more in London) after 5 years.  
 
-## What steps do you need to consider if you want to be a headteacher?
+[Find out more about teacher pay](/life-as-a-teacher/pay-and-benefits/teacher-pay). 
 
-<strong>Chris:</strong> If you want to go into senior leadership in a secondary school you’ll need to build both a curriculum role and a pastoral role, for example, head of geography and head of year into your early career.
+### Additional payments
 
-<strong>Elaine:</strong> I've got curriculum and pastoral experience and I also did school timetabling. As a headteacher you need experience of everything to get to the top.
+You may also receive additional payments on top of your regular salary for taking on extra responsibilities. These are known as teaching and learning responsibility (TLR) payments. 
 
-## How long does it take to go from an early career teacher to a headteacher?
+This could be a one-off payment for a specific project, for example, creating a timetable. Or it might be an extra responsibility attached to your job role, for example, being the head of department. 
 
-<strong>Sian:</strong> I think 8 to 10 years is realistic to become a headteacher for primary schools although you may be able to climb the career ladder a bit quicker.
+Depending on your responsibilities, you could earn up to an extra $salaries_tlr_max$. Your school will determine how much you receive.
 
-<strong>Elaine:</strong> If you’re driven you could probably get to headship in between 9 to 12 years. It took me longer but at the time it suited me to stay where I was.
+## National professional qualifications
```

Comment:
Should NPQs come before salary? Feels like they are linked to the section before?